fsbp-fix/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	bucketUtils "github.com/guardian/fsbp-tools/fsbp-fix/bucket-utils"
+)
+
+func main() {
+
+	ctx := context.Background()
+
+	fixS3_8 := flag.NewFlagSet("S3.8", flag.ExitOnError)
+
+	// fixEc2_2 := flag.NewFlagSet("EC2.2", flag.ExitOnError)
+
+	if len(os.Args) < 2 {
+		fmt.Println(len(os.Args))
+		fmt.Println(os.Args)
+		fmt.Println("expected 'S3.8' or 'EC2.2' subcommands")
+		os.Exit(1)
+	}
+
+	switch os.Args[1] {
+
+	case "S3.8":
+
+		execute := fixS3_8.Bool("execute", false, "Execute the block operation")
+		profile := fixS3_8.String("profile", "", "AWS profile to use")
+		region := fixS3_8.String("region", "", "The region of the bucket")
+		bucketCount := fixS3_8.Int("max", 100, "The maximum number of buckets to attempt to process")
+		exclusions := fixS3_8.String("exclusions", "", "Comma-separated list of buckets to skip")
+
+		fixS3_8.Parse(os.Args[2:])
+
+		if *profile == "" {
```

Comment:
In a follow-up PR, I'll introduce a new function that does all the nil value checks in one line. For now, I'm just lifting and shifting